Exercícios/Para casa/entregas/Celina-Oliveira/client.js

```diff
@@ -0,0 +1,40 @@
+const { Bank } = require('./bank');
+
+class Client {
+  name;
+  #cpf;
+  banks = [];
+
+  constructor(name, cpf) {
+    this.name = name;
+    this.#cpf = cpf;
+    this.banks = [];
+  }
+
+  get cpf() {
+    return this.#cpf; 
+}
+
+  addBank(bank)  {
+    if (this.banks.includes(bank)) {
+        console.log(`O banco ${bank.bankName} já está associado a ${this.name}.`);
+        return;
+      }
```

Comment:
Aqui acho que faltou o this.banks.push(bank); para acrescentar o bank no array. O remove eu fiz muito parecida com o add, só troquei o ++ pelo -- mas até descobrir que o remove que usei no lugar de push, tava ali só de figuração demorou... quando descobri o splice funcionou.